db/migrate/20150616183237_create_receipts.rb

```diff
@@ -9,6 +9,10 @@ def change
       t.text :item_type
       t.string :status # Order Sent -> Paid For -> In Transit -> Delivered -> Complete
       t.text :checkin_code # Allows non-logged in users to check status of their receipt
+      t.text :seller_availability # Available times seller can drop off item
+      t.text :buyer_availability # Available times buyer can drop off item.
```

Comment:
Yes. They're separate entities, both of which need to be available during their respective phase of the delivery process in order for the delivery to be successful.